### DISCUSSION TITLE
Why my execution is "laggy"?

### DISCUSSION THREAD

**Discussion by ghost** [p-org/P#405] [Q&A]
If you download and compile [this project](https://github.com/pinzaghi/themiscyra/tree/master/examples/P/Raft) and  [this dependency](https://github.com/pinzaghi/themiscyra/tree/master/examples/P/Common)

And then execute this test: `pmc POutput/netcoreapp3.1/Raft.dll -m PImplementation.TestRaftAsync.Execute -i 1 -v`

The trace shows that the execution is constantly stopping to then continue some more steps and finally stop again. This continues forever.

Is it possible that the model checking tool is having a hard time to find the next step?

**Comment by ankushdesai** [comment]
Can you try this with the new P version? and report back again?
